domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/StatementOfMeans.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class StatementOfMeans {
+    private final ResidenceType residenceType;
+    private final Dependant dependant;
+    private final Employment employment;
+    private final List<BankAccount> bankAccounts;
+    private final List<Debt> debts;
+    private final List<Income> incomes;
```

Comment:
In the screen `What regular income do you receive?` (https://hmcts-cmc-prototype.herokuapp.com/prototypes/prototype-jun-2018/defendant/task-list/owe-all/statement/income) the type `other` is somewhat structured slightly differently:
- There is an additional field to capture called `Source of income`
- There could be more than one source of income per type, hence we probably will want to model this as a `List<List<Income>>`.

Reply:
if selected others, the string entered by user will be used as type. May be like `Others: Trading`